cmd/podman/images/build.go

```diff
@@ -90,12 +90,20 @@ func init() {
 		Command: imageBuildCmd,
 		Parent:  imageCmd,
 	})
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: imageBuildCmd,
+		Parent:  buildxCmd,
```

Comment:
You have to create a separate command struct for buildx build. Do not reuse the imageBuildStruct. Cobra is not really designed to use the same command on several subcommands. Right now the use line in `podman image build --help` will show `podman buildx build [options] [CONTEXT]` instead of `podman image build [options] [CONTEXT]`. I think this is causing the other completion test to fail.